### PR TITLE
Update volcano-admission secret when it already exists 

### PR DESCRIPTION
**Pull request by Monokaix**
fix https://github.com/volcano-sh/volcano/issues/3645

### CODE REVIEW

**Comment by Monokaix** [comment]
cc @lowang-bh @hwdef

**Review by lowang-bh — COMMENTED** [review]
/lgtm

File: installer/dockerfile/webhook-manager/gen-admission-secret.sh [hunk 1]
```diff
@@ -99,13 +99,29 @@ function createSecret() {
...
+function patchSecret() {
+  TLS_KEY=$(base64 < ${CERTDIR}/server.key | tr -d '\n')
+  TLS_CRT=$(base64 < ${CERTDIR}/server.crt | tr -d '\n')
+  CA_CRT=$(base64 < ${CERTDIR}/ca.crt | tr -d '\n')
+
+  cat <<EOF > patch.json
+[
+  {"op": "replace", "path": "/data/tls.key", "value": "$TLS_KEY"},
+  {"op": "replace", "path": "/data/tls.crt", "value": "$TLS_CRT"},
+  {"op": "replace", "path": "/data/ca.crt", "value": "$CA_CRT"},
```
Comment (hwdef) [inline installer/dockerfile/webhook-manager/gen-admission-secret.sh:111]:
```suggestion
  {"op": "replace", "path": "/data/ca.crt", "value": "$CA_CRT"}
```
Will this comma cause an error?

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**